app/services/preserved_object_handler.rb

```diff
@@ -261,19 +271,51 @@ def update_pc_invalid_moab
       pres_copy = PreservedCopy.find_by!(preserved_object: pres_object, endpoint: endpoint) if pres_object
       # FIXME: what if there is more than one associated pres_copy?
       update_status(pres_copy, PreservedCopy::INVALID_MOAB_STATUS)
-      update_pc_audit_timestamps(pres_copy, true, false)
+      update_pc_audit_timestamps(pres_copy, false)
       update_db_object(pres_copy)
     end
     handler_results.remove_db_updated_results unless transaction_ok
   end
 
-  def update_pc_unexpected_version(pres_copy, pres_object, new_status, ran_moab_validation)
+  def check_status_if_not_ok_or_unexpected_version(pres_copy, found_expected_version)
+    # if pres_copy.status != PreservedCopy::OK_STATUS
```

Comment:
I think this comment can go.  I think the unless reads clearly.

Reply:
yeah, that was a commented out line from a prior iteration, not an intentional comment, will remove